test/trino.test.ts

```diff
@@ -25,8 +25,8 @@ describe('TrinoFormatter', () => {
   supportsComments(format);
   supportsCreateTable(format);
   supportsDeleteFrom(format);
-  supportsStrings(format, ["''", "X''", "U&''"]);
-  supportsIdentifiers(format, ['""']);
+  supportsStrings(format, TrinoFormatter.stringTypes);
+  supportsIdentifiers(format, ['""', '``']);
```

Comment:
Trino doesn't support `` `..` `` quoted identifiers.

See the comment in wiki: https://github.com/sql-formatter-org/sql-formatter/wiki/identifiers